packages/sources/synthetix-feeds/src/abi/USDeUSDAbi.json

```diff
@@ -0,0 +1,239 @@
+[
```

Comment:
Do we usually put the full thing here or only what we need? 
Do we need the constructor and error?

Reply:
You can have only the methods that are actually used, no need to paste everything. The drawback is that if in the future we request more methods from a contract, we would need to update the ABI file as well

Reply:
Sounds like it is better to put the full thing in anyway?